<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtChart</class>
 <widget class="QMainWindow" name="QtChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtChart</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>79</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="labViewSize">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>37</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Graphics View坐标，左上角总是（0,0），宽度=，长度=</string>
       </property>
      </widget>
      <widget class="QLabel" name="labSceneRect">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>37</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>QGraphicsView::sceneRect()</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="MyGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
